habitat-lab/habitat/articulated_agents/humanoids/README.md

```diff
@@ -32,3 +32,29 @@ You can test a simple demo where the humanoid moves their arms at random using:
 ```
 python examples/interactive_play.py --never-end --disable-inverse-kinematics --control-humanoid --cfg benchmark/rearrange/play_human.yaml
 ```
+
+## Controlling the humanoid
+
+While you can control the humanoid avatars by directly setting joint rotations, you may be interested in easily generating realistic humanoid motion. We provide a set of controllers that convert high level commands into low-level controls. We currently provide two controllers, as described below:
+
+### HumanoidRearrangeController
+
+The [HumanoidRearrangeController](../../articulated_agent_controllers/humanoid_rearrange_controller.py) controller, designed to drive the humanoid to navigate around a scene, or pick and place objects. This controller is used for the Social Navigation and Social Rearrangement tasks. It allows to generate walking motions, that drive the agent to a particular object position, and reaching motions, which change the agent pose so that the right or left hand reach a specific coordinate. You can test the controller by running:
```

Comment:
```suggestion
The [HumanoidRearrangeController](../../articulated_agent_controllers/humanoid_rearrange_controller.py) controller, designed to drive the humanoid to navigate around a scene, or pick and place objects. This controller is used for the Social Navigation and Social Rearrangement tasks. It allows generation of walk cycle animations, that drive the agent to particular object positions, and reaching motions, which change the agent pose so that the right or left hand reaches a specific realtive position. You can test the controller by running:
```